packages/account/src/Sections/Profile/PhoneVerification/confirm-phone-number.tsx

```diff
@@ -5,23 +5,31 @@ import { Localize, localize } from '@deriv/translations';
 import { Input } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 
-const ConfirmPhoneNumber = observer(() => {
+type TConfirmPhoneNumber = {
+    setOtpVerification: (value: { show: boolean; phone_verification_type: string }) => void;
+};
+
+const ConfirmPhoneNumber = observer(({ setOtpVerification }: TConfirmPhoneNumber) => {
     const { client } = useStore();
     const { account_settings } = client;
     const phoneNumber = account_settings.phone || '';
+    const handleSubmit = (phone_verification_type: string) => {
+        setOtpVerification({ show: true, phone_verification_type });
+    };
+
     return (
         <PhoneVerificationCard>
             <Text bold>
                 <Localize i18n_default_text='Confirm your phone number' />
             </Text>
             <Input label={localize('Phone number')} value={phoneNumber} />
             <div className='phone-verification__card--buttons_container'>
-                <Button variant='secondary' color='black' fullWidth size='lg'>
+                <Button variant='secondary' color='black' fullWidth size='lg' onClick={() => handleSubmit('SMS')}>
                     <Text bold>
                         <Localize i18n_default_text='Get code via SMS' />
                     </Text>
                 </Button>
-                <Button color='black' fullWidth size='lg'>
+                <Button color='black' fullWidth size='lg' onClick={() => handleSubmit('WhatsApp')}>
```

Comment:
let's not put a hard-coded value in the code (SMS & WhatsApp
)

Reply:
will copy the constant file from Utkarsha's PR 🙏 https://github.com/suisin-deriv/deriv-app/pull/86/files